.github/workflows/build.yaml

```diff
@@ -9,19 +9,19 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+
       - name: Setup LXD
-        uses: whywaita/setup-lxd@v1
-        with:
-          lxd_version: latest/stable
+        uses: canonical/setup-lxd@main
```

Comment:
JFYI, it should speed up significantly after merging: https://github.com/canonical/setup-lxd/pull/4